features/cellular/framework/device/CellularStateMachine.cpp

```diff
@@ -676,6 +663,18 @@ void CellularStateMachine::set_cellular_callback(mbed::Callback<void(nsapi_event
     _event_status_cb = status_cb;
 }
 
+bool CellularStateMachine::check_is_target_reached()
+{
+    tr_debug("check_is_target_reached(): target state %s, _state: %s, _cb_data.error: %d, _event_id: %d,_is_retry: %d", get_state_string(_target_state), get_state_string(_state), _cb_data.error, _event_id, _is_retry);
+
+    if ((_target_state == _state && _cb_data.error == NSAPI_ERROR_OK && !_is_retry) || _event_id == STM_STOPPED) {
+        tr_debug("Target state reached: %s, _cb_data.error: %d, _event_id: %d", get_state_string(_target_state), _cb_data.error, _event_id);
```

Comment:
This is almost same as above trace. And should this be tr_info rather than tr_debug?

Reply:
I'll remove, this was left while testing with greentea.